src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -62,8 +62,8 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
             Have a question?{" "}
             <Text style={{ textDecorationLine: "underline" }} onPress={() => this.handleReadOurAuctionFAQsTap()}>
               Read our auction FAQs
-            </Text>{" "}
-            or{" "}
+            </Text>
+            . or{" "}
```

Comment:
shouldn't `or` become `Or` now?

Reply:
I would assume so, yes. Shall we make this change and get it in the beta?

```suggestion
            . Or{" "}
```